protocol/x/clob/keeper/deleveraging.go

```diff
@@ -236,13 +236,37 @@ func (k Keeper) OffsetSubaccountPerpetualPosition(
 				deltaQuantums = new(big.Int).Set(deltaQuantumsRemaining)
 			}
 
+			// Fetch delta quote quantums. Calculated at bankruptcy price for standard
+			// deleveraging and at oracle price for final settlement deleveraging.
+			deltaQuoteQuantums, err := k.getDeleveragingQuoteQuantumsDelta(
+				ctx,
+				perpetualId,
+				liquidatedSubaccountId,
+				deltaQuantums,
+			)
```

Comment:
errors returned from bankruptcy price calculation were previously caught within ProcessDeleveraging and recovered from in OffsetSubaccountPerpetualPosition, continuing onto the next iteration without breaking. I am opting to continue this pattern, but will log a debug msg